File: 1highlevlsdk/qrtcroom/src/main/java/com/niucube/absroom/ForbiddenMicSeatMsg.java
package com.niucube.absroom;
//禁言消息
public class ForbiddenMicSeatMsg {

    public String uid = "";
    public boolean isForbidden = false;
    public String msg = "";
    public ForbiddenMicSeatMsg(){}
    public ForbiddenMicSeatMsg(String uid, boolean isForbidden, String msg) {
        this.uid = uid;
        this.isForbidden = isForbidden;
        this.msg = msg;
    }
}

File: 1highlevlsdk/qrtcroom/src/main/java/com/niucube/absroom/ForbiddenMicSeatMsg.kt
package com.niucube.absroom

//禁言消息
class ForbiddenMicSeatMsg {
    var uid = ""
    var isForbidden = false
    var msg = ""

    constructor() {}
    constructor(uid: String, isForbidden: Boolean, msg: String) {
        this.uid = uid
        this.isForbidden = isForbidden
        this.msg = msg
    }
}